text/0517-io-os-reform.md

```diff
@@ -1370,6 +1370,91 @@ The contents of `std::io::net` submodules `tcp`, `udp`, `ip` and
 the other modules are being moved or removed and are described
 elsewhere.
 
+#### InetAddr
+
+The composition of an `IpAddr` and a port. It has the following interface:
+
+```rust
+impl InetAddr {
```

Comment:
Yeah. I don't know what arguments a `new` function would take to make it work for both address types.